client/data/marketplace/use-wpcom-plugins-query.ts

```diff
@@ -37,6 +37,16 @@ const fetchWPCOMPlugins = ( type: Type, searchTerm?: string, tag?: string ) => {
 	);
 };
 
+export const getWPCOMPluginsQueryParams = (
+	type: Type,
+	searchTerm?: string,
+	tag?: string
+): [ QueryKey, QueryFunction< { results: Array< { [ plugin: string ]: Plugin } > } > ] => {
+	const cacheKey = getCacheKey( type + searchTerm + tag );
```

Comment:
There should be some separator between the parts of the key, so that they are not glued together as `typesearchtermtag`. That can be ambiguous.